cloud/administration/resource_access.md

```diff
@@ -0,0 +1,43 @@
+---
+id: ram
+title: Granting users rights on resources
+---
+
+Resource access rules allow you to grant users the right to see specific [resources](../resources/glossary.md#resource).
+
+By default, all users can see all resources. This is because a default access rule called "ALL" is created automatically when the platform is installed.
+
+Administrators can create more specific rules to determine which users can see which resources. Only administrators can access the **Resource Acces Management** screen.
+
+You can create as many rules as you want. If a user is mentioned in two access rules with different rights, the more permissive rule will take precedence.
+
+## Granting users rights
+
+If you don't want all users to see all resources, you need to:
+
+1. Edit the default rule called "ALL" so that it grants the right to see all resources only to you (or to all administrators). Remember to rename the rule.
```

Comment:
On est en train de faire un changement pour garantir que tous les admins aient toujours accès à toutes les ressources (hors RAM)
J'ajouterais donc une note pour dire qu'un Admin a quoiqu'il arrive toujours l'accès à toutes les ressources.


Au final, je trouve ça un peu bizarre. Moi je mettrais "1. Go to **Administration > ACL > Resource Access Management** 2. Delete the rule 3. Create the rules que vous voulez"

Reply:
c'ets ce que j'avais fait mais je sais plus qui m'a dit de pas mettre ça